source/backend/tests/unit/api/Controllers/Acquisition/AgreementControllerTest.cs

```diff
@@ -0,0 +1,74 @@
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using MapsterMapper;
+using Moq;
+using Pims.Api.Areas.Acquisition.Controllers;
+using Pims.Api.Models.Concepts.AcquisitionFile;
+using Pims.Api.Models.Concepts;
+using Pims.Api.Services;
+using Pims.Core.Test;
+using Pims.Dal.Entities;
+using Pims.Dal.Security;
+using Xunit;
+
+namespace Pims.Api.Test.Controllers
+{
+    [Trait("category", "unit")]
+    [Trait("category", "api")]
+    [Trait("group", "acquisition")]
+    [ExcludeFromCodeCoverage]
+    public class AgreementControllerTest
+    {
+        private Mock<IAcquisitionFileService> _service;
+        private AgreementController _controller;
+        private IMapper _mapper;
+
+        public AgreementControllerTest()
```

Comment:
not sure this test file is necessary. All the tests below are just executing the 'happy path'. Also, the controller does not seem to have any logic worth testing.

Reply:
@devinleighsmith has requested this in the past to keep the test coverage compliance. It you think it's something not worth doing please bring it up in the next 'Continuous development' meeting.